plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -223,4 +247,52 @@ def stub_add_changeset
+    let(:jenkins_xml_configured_with_updated_desc) { "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project>\n  <actions/>\n  <description>\n#### SAMSON DESCRIPTION STARTS ####\nFollowing text is generated by Samson. Please do not edit manually.\nThis job is triggered from following Samson projects and stages:\n* Project - Staging\n* Project - test_stage2\nBuild Parameters starting with SAMSON_ are updated automatically by Samson. Please disable automatic updating of this jenkins job from the above mentioned samson projects before manually editing build parameters or description.\n#### SAMSON DESCRIPTION ENDS ####</description>\n  <keepDependencies>false</keepDependencies>\n  <properties>\n    <com.sonyericsson.rebuild.RebuildSettings plugin=\"rebuild@1.25\">\n      <autoRebuild>false</autoRebuild>\n      <rebuildDisabled>false</rebuildDisabled>\n    </com.sonyericsson.rebuild.RebuildSettings>\n    <hudson.model.ParametersDefinitionProperty>\n      <parameterDefinitions>\n        <hudson.model.StringParameterDefinition>\n          <name>SAMSON_buildStartedBy</name>\n          <description>this is sample desc</description>\n          <defaultValue>default value</defaultValue>\n        </hudson.model.StringParameterDefinition>\n        <hudson.model.StringParameterDefinition>\n          <name>SAMSON_emails</name>\n          <description>default value 3</description>\n          <defaultValue>default value 2</defaultValue>\n        </hudson.model.StringParameterDefinition>\n      <hudson.model.StringParameterDefinition><name>SAMSON_originatedFrom</name><description>desc buildStartedBy</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_commit</name><description>desc buildStartedBy</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_tag</name><description>desc buildStartedBy</description><defaultValue/></hudson.model.StringParameterDefinition><hudson.model.StringParameterDefinition><name>SAMSON_deployUrl</name><description>desc buildStartedBy</description><defaultValue/></hudson.model.StringParameterDefinition></parameterDefinitions>\n    </hudson.model.ParametersDefinitionProperty>\n  </properties>\n  <scm class=\"hudson.scm.NullSCM\"/>\n  <canRoam>true</canRoam>\n  <disabled>false</disabled>\n  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>\n  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>\n  <triggers/>\n  <concurrentBuild>false</concurrentBuild>\n  <builders/>\n  <publishers/>\n  <buildWrappers/>\n</project>\n" }
+    before do
+      stub_crumb
+      stub_job_detail
+      deploy.stage.jenkins_build_params = true
+    end
+
+    it "adds description and build params to job configuration" do
```

Comment:
idk how to make this better, just my perspective:
no real assertions on any state / outcome here ... I would be scared to have to add new tests / understand why a test is failing (returning 0) ...